cmd/brimcap/migrate/command.go

```diff
@@ -300,6 +300,10 @@ func (m *migration) removeSpace() error {
 }
 
 func (m *migration) abort() {
-	m.conn.PoolDelete(context.Background(), m.poolID)
-	os.Remove(m.brimcapEntry)
+	if err := m.conn.PoolDelete(context.Background(), m.poolID); err != nil {
```

Comment:
Pass the context from `run`?

Reply:
No, because if run was cancelled then this would cancel the request.